src/telemetry/graph.jsx

```diff
@@ -26,7 +26,10 @@ export default class TelemetryContainer extends React.Component {
         return;
       }
 
-      this.graphTitleLink.setAttribute('href', telemetryUrl);
+      // add processType parameter and overwrite default value
```

Comment:
I don't think you need this comment since it is clear from looking at the code what is going on :)

Reply:
Removed!